fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1445,6 +1447,9 @@ public String getConnectorSinkCompressionCodec() {
     @VarAttr(name = ENABLE_MATERIALIZED_VIEW_REWRITE_PARTITION_COMPENSATE, flag = VariableMgr.INVISIBLE)
     private boolean enableMaterializedViewRewritePartitionCompensate = true;
 
+    @VarAttr(name = ENABLE_FORCE_RULE_BASED_MV_REWRITE)
+    private boolean enableForceRuleBasedMvRewrite = false;
```

Comment:
Can this be true by default?

Reply:
I think we can do it in next big version.